core/management/commands/prepare_dispatch.py

```diff
@@ -0,0 +1,79 @@
+import calendar
+import datetime
+from itertools import groupby
+from django.core.management import BaseCommand
+from django.utils import timezone
+from django.contrib.humanize.templatetags.humanize import apnumber
+
+from core.models import Event
+import click
+
+
+def generate_html_content(event_list):
+    result = []
+    for event in event_list:
+        city = event.city
+        url = event.eventpage.url
+        html = "<a href='https://djangogirls.org/%s'>%s</a>" % (url, city)
+        result.append(html)
+    return result
+
+
+class Command(BaseCommand):
+    help = 'Generate "Next events" section for the Dispatch.'
+
+    def handle(self, *args, **options):
+
+        now = timezone.now()
+        raw_dispatch_date = click.prompt(click.style("What is the date of the previous Dispatch? (Format: YYYY-MM-DD)", bold=True, fg='yellow'))
+        dispatch_date = datetime.datetime.strptime(raw_dispatch_date, "%Y-%m-%d").replace(tzinfo=datetime.timezone.utc)
+
+        # Get the events that happened since the last Dispatch.
+
+        click.echo(click.style("PREVIOUS EVENTS", bold=True))
+
+        def done_since_last_dispatch(event):
+            # Sadly, ApproximateDate won't let us do that in SQL :(
```

Comment:
Hm, I think you can actually do it, but you have to do it this way? 
```Event.objects.filter(date__gt=datetime.datetime(2016,3,1).strftime("%Y-%m-%d"), date__lt=datetime.datetime(2016,5,1).strftime("%Y-%m-%d"))```

This is like in the `future` method on `EventManager` class in `models.py`